Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -56,36 +56,6 @@ If you want to disassemble an application from the App Store, remove the Fairpla
 
 In this section the term "app binary" refers to the Macho-O file in the application bundle which contains the compiled code, and should not be confused with the application bundle - the IPA file. See section "[Exploring the App Package](0x06b-basic-security-testing.md#exploring-the-app-package "Exploring the App Package")" in chapter "Basic iOS Security Testing" for more details on the composition of IPA files.
 
```

Comment:
General comment: could you please remove all images that are not being used anymore or were relocated, such as: `Document/Images/Chapters/0x05c/Ghidra_import_binary.png`. Thank you!

Reply:
Fixed.